clay/claycli
# Delete docusaurus favicon and footer logo

# Step to test
- Go to the `website` folder
- Install the dependencies using `npm install`
- Use `npm start`
- Go to [localhost](http://localhost:3000/claycli/docs/cli.html)
- Check that the favicon doesn't exist

Note: We currently don't have the final favicon that will be on the page
